examples/docker-compose/docker-compose.yml

```diff
@@ -64,6 +64,11 @@ services:
     image: docker.io/specterops/bloodhound:${BLOODHOUND_TAG:-latest}
     environment:
       - bhe_disable_cypher_qc=${bhe_disable_cypher_qc:-false}
+      ### Add additional environment variables you wish to use here.
```

Comment:
This seemed like the best way to help users know how to get their environment variables to be respected within the BloodHound container (without reading about it in the Wiki)